PennMobile/src/main/java/com/pennapps/labs/pennmobile/adapters/HomeAdapter.kt

```diff
@@ -164,39 +143,32 @@ class HomeAdapter(
             }
 
             else -> {
```

Comment:
Why don't the DiningCell, CalendarCell, and LaundryCell get their own ViewHolders? Although it's less code since they share the same views (for now), I don't think it's a good idea to blend all of these into one. Ultimately, they are different things and may have different needs in the future.

Maybe better is to make a base holder (e.g., HomeBaseHolder) that all of the other ViewHolders in the home page inherent from. Then, in this case you can basically have a "HomeDiningHolder" that just inherits from HomeBaseHolder without any fields of its own. IMO things are more natural this way.